cmd/dagger/run.go

```diff
@@ -46,7 +59,7 @@ func Run(cmd *cobra.Command, args []string) {
 	}()
 
 	listenPort := <-listening
-	os.Setenv("DAGGER_SESSION_URL", fmt.Sprintf("http://localhost:%s", listenPort))
+	os.Setenv("DAGGER_SESSION_URL", fmt.Sprintf("http://localhost:%s/%s", listenPort, randPath))
```

Comment:
I'd prefer we use `uuid.NewRandom()` from `"github.com/google/uuid"` or similar. It's less code and more cryptographically secure, which we may as well go for since we don't need to optimize for performance here or anything like that.